src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/RazorFeedbackDiagnosticFileProvider.cs

```diff
@@ -57,7 +57,9 @@ public IReadOnlyCollection<string> GetFiles()
                 return Array.Empty<string>();
             }
 
+#pragma warning disable VSTHRD110 // Observe result of async calls
```

Comment:
Upgrading the Roslyn package versions resulted in this warning so I ended up suppressing it. If something else should be done here instead, please let me know.

Reply:
Instead of suppressing do `_ =`